src/main/kotlin/dev/proxyfox/command/menu/CommandMenu.kt

```diff
@@ -0,0 +1,39 @@
+package dev.proxyfox.command.menu
+
+import dev.proxyfox.command.CommandContext
+import dev.proxyfox.command.ScreenBuilder
+
+public abstract class CommandMenu {
+    public var default: Pair<String, ScreenBuilder>? = null
+    public var active: CommandScreen? = null
+    private var screens: HashMap<String, ScreenBuilder> = hashMapOf()
+
+    public fun default(name: String = "default", action: ScreenBuilder) {
+        screens[name] = action
+        default = Pair(name, action)
+    }
+
+    public operator fun String.invoke(action: ScreenBuilder) {
+        screens[this] = action
+        if (default == null)
+            default = Pair(this, action)
+    }
+
+    internal fun setScreen(name: String) {
+        val screen = createScreen(name)
+        screens[name]!!(screen)
+        screen.init()
+        active = screen
+    }
```

Comment:
This seems potentially fragile, tho I'm not sure if there's a better solution here.

Ideally, some kind of hard reference to prevent bugs via dev error.